API-CHANGELOG.md

```diff
@@ -83,23 +83,29 @@ The [commandline](https://xrpl.org/docs/references/http-websocket-apis/api-conve
 
 The `network_id` field was added in the `server_info` response in version 1.5.0 (2019), but it is not returned in [reporting mode](https://xrpl.org/rippled-server-modes.html#reporting-mode). However, use of reporting mode is now discouraged, in favor of using [Clio](https://github.com/XRPLF/clio) instead.
 
-## XRP Ledger server version 2.2.0
-
-The following is a non-breaking addition to the API.
+## XRP Ledger server version 2.3.0
 
-- The `feature` method now has a non-admin mode for users. (It was previously only available to admin connections.) The method returns an updated list of amendments, including their names and other information. ([#4781](https://github.com/XRPLF/rippled/pull/4781))
+[Version 2.3.0](https://github.com/XRPLF/rippled/releases/tag/2.3.0) was released on Nov 25, 2024.
 
-### Breaking change in 2.3
+### Breaking changes in 2.3.0
```

Comment:
```suggestion
### Breaking changes
```
2.3.0 is already specified in the header, I don't think it needs to be specified here as well

Reply:
We double-specified it so that the anchor links are unique and thus linkable. Example: https://github.com/XRPLF/rippled/blob/develop/API-CHANGELOG.md#breaking-changes-in-111

If it's just `Breaking changes` then the anchor link becomes ambiguous